components/exercises/chapter-5-oscar/Everything/every.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Checks if every element in an array satisfies a given condition.
+ * @param array - The array to check.
+ * @param callback - The condition function to apply to each element.
+ * @returns `true` if every element satisfies the condition, `false` otherwise.
+ */
+export const every = (
+  arr: number[],
```

Comment:
why limit only to numbers? I think this function should be able to handle arrays of any type

Reply:
I limited to numbers because that is what I am using it for

Reply:
I've got the following error if I add strings aside numbers: 
The error "Operator '>' cannot be applied to types 'string | number' and 'number'.ts(2365)". Therefore the solution was TypeScript's generic type parameter (<T>) to make the function more flexible and allow it to work with arrays containing elements of different types.